examples/snippets/modules/ROOT/examples/base-images-override/build.sbt

```diff
@@ -3,8 +3,9 @@ import sbt.Keys._
 
     //tag::docs-projectSetup-example[]
 lazy val sampleApp = (project in file("."))
+    .enablePlugins(CloudflowApplicationPlugin)
     .settings(
-      cloudflowAkkaBaseImage := "myRepositoryUrl/myRepositoryPath:2.0.10-cloudflow-akka-2.6.6-scala-2.12",
+      cloudflowDockerBaseImage := "myRepositoryUrl/myRepositoryPath:2.0.10-cloudflow-akka-2.6.6-scala-2.12",
```

Comment:
I'm not sure what is going on here ...

Reply:
Avoid using the deprecated setting in the example.